code/datums/components/tippable.dm

```diff
@@ -68,7 +68,7 @@
 
 	if(is_tipped)
 		INVOKE_ASYNC(src, .proc/try_untip, source, user)
-	else if(LAZYACCESS(modifiers, RIGHT_CLICK))
+	else
 		INVOKE_ASYNC(src, .proc/try_tip, source, user)
 
 	return COMPONENT_CANCEL_ATTACK_CHAIN
```

Comment:
@JohnFulpWillard This needs to get changed to `return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN`, I'm not sure what this value is set to but it's the wrong enum

Reply:
I THOUGHT I DID IT WTF

Reply:
lmao, lol said the scorpion